TDDWithURMAndKotlin/part2_increment.md

```diff
@@ -0,0 +1,157 @@
+# TDD With URM and Kotlin - increment
+
+In the previous article we mentioned a way to start writing our implementation of URM using TDD. We also mentioned a lot of questions that came up while writing the `zero` function.
```

Comment:
Shouldn't you add a link to the previous article? 🤔

Reply:
Yes BUT at the moment this was written no actual article was published